tests/common/devices.py

```diff
@@ -1044,6 +1045,86 @@ def get_namespace_from_asic_id(self, asic_id):
             return DEFAULT_NAMESPACE
         return "{}{}".format(NAMESPACE_PREFIX, asic_id)
 
+class K8sMasterHost(AnsibleHostBase):
+    """
+    @summary: Class for Ubuntu KVM that hosts Kubernetes master
+
+    For running ansible module on the K8s Ubuntu KVM host
+    """
+
+    def __init__(self, ansible_adhoc, hostname, ip_addr, k8s_user, k8s_passwd):
+        '''Initialize an object for interacting with Ubuntu KVM using ansible modules
+        
+        Args:
+            ansible_adhoc (): The pytest-ansible fixture
+            hostname (string): hostname of the Ubuntu KVM
+            k8s_user (string): Username for accessing the K8s VM CLI interface
+            k8s_passwd (string): Password for the k8s_user 
+        '''
+        self.k8s_user = k8s_user
+        self.k8s_passwd = k8s_passwd
+        self.hostname = hostname
+        self.ip_addr = ip_addr
+        AnsibleHostBase.__init__(self, ansible_adhoc, hostname)
```

Comment:
>AnsibleHostBase [](start = 8, length = 15)

use `super` ?

Reply:
Similar implementations of `AnsibleHostBase` class (`EosHost`, `OnyxHost`, etc.) use `AnsibleHostBase.__init__ `
Is `super` advisable for other implementations of `AnsibleHostBase` class too?

Reply:
ref: [https://stackoverflow.com/a/576183/2514803](https://stackoverflow.com/a/576183/2514803)
It is recommended.

---
In reply to: [513072760](https://github.com/Azure/sonic-mgmt/pull/2382#discussion_r513072760) [](ancestors = 513072760)

Reply:
Thanks for the suggestion, I changed K8sMasterHost class to use `super` instead of explicitly calling `AnsibleHostBase`